doc/stdlib/file.rst

```diff
@@ -337,7 +337,7 @@
 
    .. Docstring generated from Julia source
 
-   ``@__FILE__`` expands to a string with the absolute file path of the file containing the macro. Returns ``nothing`` if run from a REPL or an empty string if evaluated by ``julia -e <expr>``. Alternatively see :data:`PROGRAM_FILE`.
+   ``@__FILE__`` expands to a string with the absolute file path of the file containing the macro. Returns ``nothing`` if run from a REPL or an empty string if evaluated by ``julia -e <expr>``\ . Alternatively see ```PROGRAM_FILE`` <:data:`PROGRAM_FILE`>`_\ .
```

Comment:
This change sneaked in upon running `make julia-genstdlib` . I presume it is correct.